README.adoc

```diff
@@ -81,27 +81,25 @@ appsody repo list
 You see an output similar to the following example:
 [source, role='no_copy']
 ----
-appsody repo list
 NAME        URL
-*appsodyhub https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+*incubator https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
 ----
 
 Next, run the following command to add the Kabanero index:
 [role="command"]
 ----
-appsody repo add kabanero https://github.com/kabanero-io/collections/releases/download/v0.1.2/kabanero-index.yaml
+appsody repo add kabanero https://github.com/kabanero-io/collections/releases/download/v0.3.0/kabanero-index.yaml
```

Comment:
Only one problem to fix that I can see. The Collections team removed the "v" from the URL string ... so it is just `download/0.3.0/kab....` .... small change in just a few places.